qos/cometbft/endpoint.go

```diff
@@ -21,14 +24,20 @@ type endpoint struct {
 	// nil if no health check has been performed yet.
 	healthResponse *bool
 
-	// parsedBlockNumberResponse stores latest block height reported by the endpoint.
+	// chainIDResponse stores the chain ID of the endpoint.
+	chainIDResponse string
+
+	// syncedResponse stores if the endpoint is not catching up.
```

Comment:
I'm assuming this is based on the value of `sync_info. catching_up`. If so, #PUC